vars/acePushConfigToGit.groovy

```diff
@@ -7,14 +7,17 @@ void call(Map opts = [:]) {
   Map gitops = cfg.gitops ?: [:]
   String gitopsRepo = gitops.repo
   String strategy = gitops.strategy ?: 'path'
+  String helmChart = target.chart
+  String helmChartVersion = target.version
 
   if (!gitopsRepo) {
     error('[ace] No gitops repo specified, dying.')
   }
 
   print "[ace] gitops repo strategy is ${strategy}"
 
-  helmPullChart(target.chart, opts)
+
+  helmPullChart(helmChart, helmChartVersion, opts)
```

Comment:
If you put the version inside the `opts` the change would be backwards compatible. Not sure if there are other instances using the `helmPullChart` directly?

Reply:
Change to set default values so that is should now be backward compatible.

Reply:
What happens if someone puts a helmChartVersion in opts? This is where it should go I think, not as a separate option.